server/routes/html.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+const ejs = require('ejs');
+const path = require('path');
+const urlHelpers = require('auth0-extension-express-tools').urlHelpers;
+
+const config = require('../lib/config');
+
+module.exports = () => {
+  const template = `
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <title><%= config.TITLE %></title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="https://cdn.auth0.com/styleguide/4.6.13/lib/logos/img/favicon.png">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="https://cdn.auth0.com/styles/zocial.min.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.auth0.com/manage/v0.3.1672/css/index.min.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.auth0.com/styleguide/4.6.13/index.min.css" />
+    <% if (assets.style) { %><link rel="stylesheet" type="text/css" href="/app/<%= assets.style %>" /><% } %>
+    <% if (assets.version) { %><link rel="stylesheet" type="text/css" href="//cdn.auth0.com/extensions/auth0-management-api-webhooks/assets/auth0-management-api-webhooks.ui.<%= assets.version %>.css" /><% } %>
+    <% if (assets.customCss) { %><link rel="stylesheet" type="text/css" href="<%= assets.customCss %>" /><% } %>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="text/javascript" src="//cdn.auth0.com/w2/auth0-7.0.4.min.js"></script>
+    <script type="text/javascript" src="//cdn.auth0.com/manage/v0.3.1672/js/bundle.js"></script>
+    <script type="text/javascript">window.config = <%- JSON.stringify(config) %>;</script>
```

Comment:
I have concerns that this could be vulnerable to injection attacks. Not 100% either way, but commenting for a 2nd pair of eyes to look at.

Reply:
Upon further thinking, I'm pretty sure we are good here since the config isn't accepting user input. 👍